source/getting-started/devices.markdown

```diff
@@ -14,7 +14,10 @@ routers, Belkin WeMo switches, Philips Hue bridges and Sonos speakers in your ne
 [the discovery component]({{site_root}}/components/discovery.html) enabled (which is by default).
 
 See the [components overview page](/components/) to find installation instructions for your devices
-and services.
+and services.  Many components use the `sensor` platform.  Sensors need to be added into the `configuration.yaml` as `sensor:`, `sensor 2:`, `sensor 3:`, etc.  There should not be gaps in the sequence or your sensors might not load correctly.
+<p class='note'>
+Most components that support multiple entries within the `configuration.yaml` follow the `component:`, `component 2:` format.
```

Comment:
Backticks do not work inside a note because Markdown parses this as HTML. Please use `<code>` and `</code>`